yarn.lock

```diff
@@ -1608,6 +1612,26 @@ escodegen@^1.8.1:
   optionalDependencies:
     source-map "~0.6.1"
 
+eslint-plugin-ember@^5.0.3:
+  version "5.2.0"
+  resolved "https://registry.yarnpkg.com/eslint-plugin-ember/-/eslint-plugin-ember-5.2.0.tgz#fa436e0497dfc01d1d38608229cd616e7c5b6067"
+  dependencies:
+    ember-rfc176-data "^0.3.3"
+    snake-case "^2.1.0"
+
+eslint-plugin-ghost@^0.0.27:
```

Comment:
Whoops just saw this. 

You need to re-generate the file or re-add the dep correctly, because you initially added `^0.0.27`.

Reply:
Ohh wait. will check.

Reply:
@kirrg001 done.